src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/LocalDebuggerCommandValueProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using System;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties
+{
+    [ExportInterceptingPropertyValueProvider("LocalDebuggerCommand", ExportInterceptingPropertyValueProviderFile.UserFileWithXamlDefaults)]
+    internal class LocalDebuggerCommandValueProvider : InterceptingPropertyValueProviderBase
+    {
+        public const string DefaultCommand = "";
+
+        public override Task<string> OnGetEvaluatedPropertyValueAsync(string evaluatedPropertyValue, IProjectProperties defaultProperties)
+        {
+            var command = evaluatedPropertyValue;
+
+            // If the existing command is the empty string, then a custom command has not been set. We therefore want to return dotnet here,
+            // and return exec + exe + args in the LocalDebuggerCommandArguments interceptor.
+            if (command == DefaultCommand)
+                command = "dotnet.exe";
```

Comment:
I think we'll want have a property that represents the location of this thing. Clearly dotnet.exe isn't going to be on the path all the time.